internal/output/combined.go

```diff
@@ -172,13 +172,13 @@ func CompareTo(c *config.Config, current, prior Root) (Root, error) {
 
 	// preserve the summary from the original run
 	currentProjects := make(map[string]Project)
-	for _, p := range prior.Projects {
+	for _, p := range current.Projects {
```

Comment:
I think `current` makes more sense here, since then the summaries sum to match the overall summaries, which are also set using `current`.